youtube_dl/extractor/peertube.py

```diff
@@ -569,15 +569,15 @@ def _real_extract(self, url):
             formats.append(f)
         self._sort_formats(formats)
 
-        full_description = self._call_api(
-            host, video_id, 'description', note='Downloading description JSON',
-            fatal=False)
+        description = video.get('description')
+        if len(description) == 250 and description.endswith('...'):
```

Comment:
it might be better to use the same condition that is used by [PeerTube's client code](https://github.com/Chocobozzz/PeerTube/blob/d55c466e1caff9082e8ca1734fbf079ac92e12b4/client/src/app/%2Bvideos/%2Bvideo-watch/video-watch.component.html#L216)

Reply:
The first check is just "is the button clicked", the second is >= 250.

From my test max is always 250, not above and it ends with `...` but I changed to >= 250 as you wanted in the last commit.